formats/completed_transaction/frontend/examples/completed_transaction.json

```diff
@@ -0,0 +1,52 @@
+{
+  "details": {
+    "external_related_links": [
+
+    ],
+    "promotion_choice": {
```

Comment:
When a government user selects "none" in the publisher, can this block be omitted altogether?